src/models/sql/sql.jwt.sql

```diff
@@ -0,0 +1,9 @@
+-- Contains JWT refresh tokens
+CREATE TABLE `JWTRefreshTokens` IF NOT EXISTS (
+    customerID INT PRIMARY KEY NOT NULL,
+    refreshToken BINARY(32), /*TODO: is 256 bytes ok?*/
+    expiration TIMESTAMP,
+    FOREIGN KEY (customerID) REFERENCE customers(id),
+) ENGINE = InnoDB,
+CHARSET = utf8,
+COLLATE = utf8_spanish2_ci;
```

Comment:
What is that?
Why have you created a table to store all the refresh tokens?

Reply:
No, it's not our approach, we don't need to storage any refresh token, that's the key to success. Here we are focusing only on disable accounts which don't need to sign in again, therefore (thanks to JWT middleware second step which checks if accounts is enabled) allow us to disabled all refresh tokens without having to storage it and without using any third part library. 

If you are not sure to understand what I'm proposing please ask me directly.